code/game/machinery/cryopod.dm

```diff
@@ -310,6 +310,31 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		qdel(W)//because we moved all items to preserve away
 		//and yes, this totally deletes their bodyparts one by one, I just couldn't bother
 
+	// Suspend their bank payment
+	if(mob_occupant?.mind?.account_id)
```

Comment:
I don't think mob_occupant will ever be null

Reply:
I went too defensive.